platform/lib/utils/googleSearch.js

```diff
@@ -36,21 +36,31 @@ credentials.get('GOOGLE_CSE_API_KEY').then((key) => {
       err.message ? err.message : err);
 });
 
-async function search(query, locale, page) {
+async function search(query, locale, page, options = {}) {
   if (!API_KEY) {
     throw Error('Custom search api key not initialized! Check log for errors on startup.');
   }
 
   const startIndex = (page - 1) * PAGE_SIZE + 1;
   query = encodeURIComponent(query);
+  const hiddenQuery = options && options.hiddenQuery ? encodeURIComponent(options.hiddenQuery) : '';
```

Comment:
I don't think we need the null check here. options cannot be undefined except someone explicitly passes null. In that case we should fail.